File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseBox.scala

package org.openapitools.client.model


case class ResponseBox (
    _minLat: Double,
    _maxLat: Double,
    _minLng: Double,
    _maxLng: Double
)
object ResponseBox {
    def toStringBody(var_minLat: Object, var_maxLat: Object, var_minLng: Object, var_maxLng: Object) =
        s"""
        | {
        | "minLat":$var_minLat,"maxLat":$var_maxLat,"minLng":$var_minLng,"maxLng":$var_maxLng
        | }
        """.stripMargin
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/ResponseBox.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic

module ResponseBox = 

  //#region ResponseBox


  type ResponseBox = {
    MinLat : double;
    MaxLat : double;
    MinLng : double;
    MaxLng : double;
  }
  //#endregion
  